q2_feature_table/_normalize.py

```diff
@@ -9,8 +9,10 @@
 import biom
 
 
-def rarefy(table: biom.Table, sampling_depth: int) -> biom.Table:
-    table = table.subsample(sampling_depth, axis='sample', by_id=False)
+def rarefy(table: biom.Table, sampling_depth: int,
+           with_replacement: bool=False) -> biom.Table:
```

Comment:
```suggestion
           with_replacement: bool = False) -> biom.Table:
```

Reply:
This is a silly linting error related to the newest version of `flake8`, which is why travis is currently failing, @stevendbrown.

Reply:
To clarify, the error itself is silly, not what you wrote... this new version of `flake8` has generated a lot of busywork for us the last few days!